### ISSUE TITLE
Experimental Input System: Pen too small to correctly see

### ISSUE THREAD

**Issue by terencode** [xournalpp/xournalpp#1654]
(Please complete the following information, and then delete this line)

**Affects versions :**
 - OS: ArchLinux
 - Gnome on X11
 - libgtk 3.24.13
 - Xournal++ 1.0.16

**Bug Description**
When using the experimental input system, the pen is really small, making it barely visible. Moreover it doesn't even correspond to the thickness selected.

**To Reproduce**
1. Enable the experimental input
2. Notice your default edit cursor is replaced with a very small dot
3. Change the thickness of the pen and notice the dot doesn't account for it anyway

**Expected behavior**
Either the dot accommodates for the difference in thickness but it's still hard to see or make it more visible (crosshair? fake pen? arrow?). 

**Video**
![Peek 2019-12-05 10-57](https://user-images.githubusercontent.com/12665563/70225193-dd8b4000-174e-11ea-8268-8c8959f1dd6e.gif)

**Comment by terencode** [comment]
I see there is a 'big cursor" for pen option which seems to be good enough for my case. Maybe there should be a note about it, enable it by default ?
I guess the issue about the pen not taking the thickness into account still stands.

**Comment by LittleHuba** [comment]
Yes the big cursor is exactly for this scenario. We are of the impression that if you use your pen to write you know well enough where your pen is. For graphics tablets again we have the big cursor option.

In the long run we will allow for userprovided svgs instead of the big cursor.

**Comment by terencode** [comment]
Yeah that was hard on a tablet.
What about the point accounting for the thickness selected?

**Comment by LittleHuba** [comment]
> What about the point accounting for the thickness selected?

That is indeed an issue that needs to be solved.

**Comment by SergioQuijanoRey** [comment]
Same happens to me using an Huion hp 640. It makes the app unusable. Currently using old xournal app without this problem.
